person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
```

Comment:
- [ ] It would be better if we create a `correct_name` method for our `Person` class as mentioned by the previous code reviewer